scripts/bluesky-0MQ-proxy

```diff
@@ -1,6 +1,78 @@
 #!/usr/bin/env python3
+import threading
 import argparse
+import logging
+import sys
 
+from bluesky.callbacks.zmq import RemoteDispatcher
+
+class Dispatcher_thread(threading.Thread):
+    def __init__(self, logfile, poison_pill):
+        super(Dispatcher_thread, self).__init__()
+        self._logfile = logfile
+        self._poison_pill = poison_pill
+        self._dispatcher = None
+
+    def set(self, poison_pill):
+        self._dispatcher.set_pill(poison_pill)
+
+    def run(self):
+        self._dispatcher = RemoteDispatcher(f'localhost:{out_port}', self._poison_pill)
+        print("self._logfile")
+        print(self._logfile)
+        if self._logfile:
+            print("inside log")
+            logger = logging.getLogger('logfile')
+            hdlr = logging.FileHandler(self._logfile)
+            formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
+            hdlr.setFormatter(formatter)
+            logger.addHandler(hdlr)
+            logger.setLevel(logging.INFO) #Only write to file with logger.info or worse
```

Comment:
stylistic comment - there should be a blank space after `#`, and 2 blank spaces between the end of the code string and the comment.